src/screens/EditWordScreen/EditWordScreen.js

```diff
@@ -62,21 +58,13 @@ class EditWordScreen extends Component {
   }
 
   generateTranslation = translationList => translationList.map((word) => {
-    const deleteCurrentValue = () => this.deleteWordOrExample(word, WORD)
+    const deleteCurrentValue = () => this.deleteWordOrExample(word, EDIT_TYPES.word)
 
     return (
       <View style={styles.translationItem} key={uuidv4()}>
-        <Text
-          style={{
-            fontFamily: 'OpenSans',
-            color: 'white',
-            fontSize: 20,
-          }}
-        >
-          {word}
-        </Text>
+        <Text style={styles.itemText}>{word}</Text>
```

Comment:
`generateTranslation` & `generateExamples` looks the same, refactor needed

Reply:
Agree